test/Network/MQTT/ParserEncoderSpec.hs

```diff
@@ -0,0 +1,61 @@
+{-# LANGUAGE FlexibleInstances, MultiParamTypeClasses #-}
+
+module Network.MQTT.ParserEncoderSpec       (spec) where
```

Comment:
The Hspec convention is to make Spec with same names as real modules (i.e., the name should be either `ParserSpec` or `EncoderSpec`.)